server/src/main/java/org/elasticsearch/action/admin/cluster/reroute/ClusterRerouteResponse.java

```diff
@@ -43,6 +44,7 @@ public class ClusterRerouteResponse extends ActionResponse implements IsAcknowle
     /**
      * To be removed when REST compatibility with {@link org.elasticsearch.Version#V_8_6_0} / {@link RestApiVersion#V_8} no longer needed
      */
+    @UpdateForV9(owner = UpdateForV9.Owner.DISTRIBUTED_COORDINATION)
```

Comment:
If I am not mistaken, according to the comment above, this should be updated in `UpdateForV10` as `RestApiVersion.V_8` is not going to be removed in V9

Reply:
Oh, yes, quite right